napari/_qt/qt_viewer.py

```diff
@@ -296,9 +297,25 @@ def __init__(self, viewer: ViewerModel, show_welcome_screen: bool = False):
         # bind shortcuts stored in settings last.
         self._bind_shortcuts()
 
+        settings = get_settings()
+        self._update_dask_settings(settings.application.dask)
+
+        settings.application.events.dask.connect(self._update_dask_settings)
```

Comment:
```suggestion
        settings.application.events.dask.connect(self._update_cache_settings)
settings.application.dask.events.enabled.connect(self._update_cache_settings)
        settings.application.dask.events.cache.connect(self._update_cache_settings)        
```

Reply:
When changing the widget, the cache values were not updating, so these extra lines needed to be added (I think) because the dask settings object is now an eventedModel.